content/api/inline-seeds.apib

```diff
@@ -0,0 +1,511 @@
+
+FORMAT: 1A
+title: Automatic Inline Seeding
+description: Accept campaign stats, sanitize PII, and trigger seed messages.
+label: Deliverability
+# Group Automatic Inline Seeding API
+
+The Automatic Inline Seeding API helps customers automatically send messages from their campaigns to seed addresses. This document is for customers that are sending with SparkPost. Customers that are sending using another offering should see TODO: link here to on-prem docs.
+
+## Seed Config
+
+Seeding options that will apply to campaigns that match the provided rules.
+
++ Data Structure
+  + threshold (number) - Number of messages seen for a campaign before seed sending starts. <br /> Maximum: `1,000,000`<br /> Minimum: `1,000`
+    + Default: 10,000
+  + duration (number) - Number of minutes after threshold is met over which to spread out seed messages. <br /> Maximum: `720` (12 hours)<br /> Minimum: `20`
+    + Default: 30
+  + reset (number) - Number of hours after threshold is met until seed sending can happen again. <br /> Maximum: `168` (7 days) <br /> Minimum: `22`
+    + Default: 24
+  + match (array) - Array of campaign ids or prefixes that are eligible. Strings ending with an asterisk are treated as prefixes. <br /> Maximum: `5` entries.
+  + exclude (array) Array of campaign ids or prefixes that are NOT eligible. Strings ending with an asterisk are treated as prefixes. <br /> Maximum: `5` entries.
+
+### Get Seed Config [GET /inline-seeds/config]
```

Comment:
I think that we need to add `/v1` before `/inline-seeds/config` so that the URL appears as `/api/v1/inline-seeds/config` in the rendered docs. Same goes for all other headings.